### PR TITLE
Overflow fix

### LINKED ISSUE #36: l18n strings overflow
<Do not use this to ask about lost codes or accounts>
<More info here: https://github.com/Authenticator-Extension/Authenticator/wiki/Lost-Codes>

### PR DESCRIPTION
**Pull request by mymindstorm**
Why is it so hard to center things?

closes #36

### CODE REVIEW

**Comment by Sneezry** [comment]
The menu looks nice now. The Dropbox code input box looks not as good as previous, how about to change it back to previous version?

File: css/popup.css [hunk 1]
```diff
@@ -515,10 +515,17 @@ body {
     height: 480px;
     position: absolute;
     left: -1000px;
-    background: #EEE;
     top: 0;
 }
 
+#menuBody {
+    overflow-y: scroll;
```
Comment (Sneezry) [inline css/popup.css]: `auto` should be better here

**Comment by mymindstorm** [comment]
I think I was able to make it cooperate.

**Comment by Sneezry** [comment]
Awesome!

**Review by Sneezry — APPROVED** [review]

**Pull request merged**